src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -546,9 +546,35 @@ static void _updateText(LottieGroup* parent, LottieObject** child, float frameNo
     float spacing = text->spacing(frameNo) / scale;
     Point cursor = {0.0f, 0.0f};
     auto scene = Scene::gen();
+    int line = 0;
 
     //text string
-    while (*p != '\0') {
+    while (true) {
```

Comment:
Could you please leave a TODO comment so that we can try it by any chance later?
```
TODO: remove nested scenes.
```